src/core/navigation/navigation.view.js

```diff
@@ -66,4 +75,23 @@ export default class NavigationView extends View {
 		return this.markup.body.rows;
 	}
 
+	scroll(inner, outer) {
```

Comment:
rename inner to target, and outer to container and move container to the 1st place